src/components/ArtistTile.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Header from './components/Header';
+import Footer from './components/Footer';
+
+{
+  currentUser && (
+    <Route
+      path="/ArtistTile"
+      element={
+        <ArtistProtectedIndex
+          currentUser={currentUser}
+          artist={artist}
+        />
+      }
+    />
+  )
+}
+
```

Comment:
Same thing on this component - no need for header or footer and your route should be in App.js